projects/knora-ui/src/lib/viewer/values/text-value/text-value-as-html/text-value-as-html.component.html

```diff
@@ -0,0 +1,4 @@
+<div [innerHTML]="htmlFromKnora" class="value"></div>
```

Comment:
The html is okay for now, but the CSS I've been working on will need to be adapted to fit this readonly, non form value

Reply:
Shall we merge this PR first so you can integrate it into #43?

Reply:
yeah I think that sounds good!

Reply:
ok, great!